Fixes Stermi review (#374)
morpho-org/metamorpho: README.md

For more information about this use case, see the [Rewards](#rewards) section.

All actions that may be against users' interests (e.g. enabling a market with a high exposure) are subject to a timelock of minimum 24 hours.

> If set, the `guardian` or the `owner` can revoke the action during the timelock.

Reviewer: ```suggestion
The `owner`, or the `guardian` if set, can revoke the action during the timelock.
```

In some cases, the vault's curator or allocators may want to keep some funds "idle", to guarantee lenders that some liquidity can be withdrawn from the vault (beyond the liquidity of each of the vault's markets).

> To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having this canonical configuration:

Reviewer 2: ```suggestion
To achieve this, they can deposit in markets with `address(0)` as the oracle or the collateral, ensuring that these funds can't be borrowed. It is advised to use these canonical configuration for "idle" markets:
```

- `loanToken`: The vault's asset to be able to supply/withdraw funds.
- `collateralToken`: `address(0)` (not necessary since no funds will be borrowed on this market)
- `irm`: `address(0)` (Morpho Blue will skip the call to the IRM in this case, thus reducing the gas cost)
- `oracle`: `address(0)` (not necessary since no funds will be borrowed on this market)

> - `lltv`: `0` (not necessary since no funds will be borrowed on this market)

Reviewer 2: I'm not sure that here it's clear that you NEED at least one of collat=0 and oracle=0 otherwise people might be able to borrow your assets

Having this configuration for all idle markets will make risk management easier.

> It also ensures that no funds can be borrowed on Morpho Blue and are guaranteed to be liquid; though it won't generate interest.

Reviewer 2: sounds weird
```suggestion
This configuration also ensures that no funds deposited on these markets can be borrowed and are guaranteed to be liquid; though they won't generate interest.
```

> Having this configuration for all idle markets will make risk management easier.
> It also ensures that no funds can be borrowed on Morpho Blue and are guaranteed to be liquid; though it won't generate interest.

Reviewer 2: ```suggestion
```
